src/util.rs

```diff
@@ -44,6 +44,16 @@ pub fn history() -> web_sys::History {
     window().history().expect("Can't find history")
 }
 
+/// Convenience function to access the `web_sys::HtmlDocument`.
+pub fn html_document() -> web_sys::HtmlDocument {
+    document_to_html_document(document())
+}
+
+/// Convenience function to convert a `Document` to an `HtmlDocument`.
+fn document_to_html_document(doc: web_sys::Document) -> web_sys::HtmlDocument {
```

Comment:
I would inline `document_to_html_document` body to `html_document` because `document_to_html_document` is private one-liner and called in one place.. but yeah, it's not very important :)

Reply:
done :)